test/bindings-test.js

```diff
@@ -31,6 +32,16 @@ var MockComponent = require("./mock-component");
 
 var DefaultMap = types.DefaultMap;
 
+function afterMutation(cb) {
+	var doc = DOCUMENT();
+  var div = doc.createElement("div");
```

Comment:
Yeah, I started with copying your code snippet from Slack, which ended up having space indentations.  This and your other review comment here are addressed in #176